var/spack/repos/builtin/packages/python/package.py

```diff
@@ -148,6 +148,15 @@ def install(self, spec, prefix):
         make()
         make('install')
 
+        self.sysconfigfilename = '_sysconfigdata.py'
+        if spec.satisfies('@3.6:'):
```

Comment:
Can you add a commented URL pointing the reader to a description of the issue being addressed here?